spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/AdminApplication.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package org.springframework.cloud.dataflow.admin;
+package org.springframework.cloud.dataflow.server;
```

Comment:
I guess this class will go away once we're done refactoring the various SPI impls, but should we go ahead and rename it in the meantime?